### PR TITLE
[Tree shaking] Fix import and update ESlint rules

### PR DESCRIPTION
**Pull request by ayush547**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Some import exports needed to be fixed to allow partners to have tree shaking enabled in their apps.
Made the changes and updated the eslint config to automatically capture these issues and fix them as required.
Cleaned up extra eslintrc.js file present in Avatar/src folder of component and experimental.

### Verification

(how the change was tested, including both manual and automated tests)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Comment by tido64** [comment]
Would it make sense to split this in two? One for enabling the ESLint rule and fixing up all imports, and a separate one for configuring and enabling tree shaking?

**Comment by ayush547** [comment]
> Would it make sense to split this in two? One for enabling the ESLint rule and fixing up all imports, and a separate one for configuring and enabling tree shaking?

Agreed. Updating this PR to handle ESlint rule and fixing imports,

File: packages/components/Tabs/src/TabsItem.settings.windows.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import type { IComposeSettings } from '@uifabricshared/foundation-compose';
 import type { TabsItemType } from './TabsItem.types';
 import { tabsItemName } from './TabsItem.types';
 
+export const tabsItemSelectActionLabel = 'Select a TabsItem';
```
Comment (tido64) [inline packages/components/Tabs/src/TabsItem.settings.windows.ts:7]: Was this an artifact of rebasing? Can you revert it?

Comment (ayush547) [reply packages/components/Tabs/src/TabsItem.settings.windows.ts:7]: Had added this to fix some export issue, checked again, looks like it is no longer required. Removing it

**Review by tido64 — APPROVED** [review]

**Review by rurikoaraki — APPROVED** [review]

**Pull request merged**